src/main/java/uk/gov/dvsa/model/cvs/certificateData/RequiredStandard.java

```diff
@@ -0,0 +1,114 @@
+package uk.gov.dvsa.model.cvs.certificateData;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RequiredStandard {
+    @JsonProperty
+    private boolean prs;
+    @JsonProperty
+    private String requiredStandard;
+    @JsonProperty
+    private String refCalculation;
+    @JsonProperty
+    private String sectionNumber;
+    @JsonProperty
+    private boolean additionalInfo;
+    @JsonProperty
+    private String sectionDescription;
+    @JsonProperty
+    private int rsNumber;
+    @JsonProperty
+    private String[] inspectionTypes;
+    @JsonProperty
+    private String additionalNotes;
+
+    public RequiredStandard(boolean prs, String requiredStandard, String refCalculation, String sectionNumber, boolean additionalInfo, String sectionDescription, int rsNumber, String[] inspectionTypes, String additionalNotes) {
```

Comment:
Why is the constructor being overloaded here? This one isn't being used it seems